### PR TITLE
Colouring

### PR DESCRIPTION
**Pull request by MeikeWeiss**
I did some ToDos in ColouredComplex and resolved the problem of coloured drawing by adding SetIsEdgeColouredTwistedPolygonalComplex.

### CODE REVIEW

File: gap/ColouredComplexes/edgeColouring.gd [hunk 1]
```diff
@@ -322,7 +321,21 @@ DeclareOperation( "DrawSurfaceToTikz", [IsEdgeColouredPolygonalComplex and IsNot
...
 #! 
-#! TODO example
+#! As an example consider the coloured Tetrahedron:
+#! <Alt Only="TikZ">
+#!   \input{_TIKZ_ColouredTetrahedron.tex}
+#! </Alt>
+#! @BeginExampleSession
+#! gap> colTetra := EdgeColouredPolygonalComplex(Tetrahedron(), [1,2,3,3,2,1]);;
+#! @EndExampleSession
+#! For example, is edge 1 incident to the vertices 1 and 2, incident to the faces 1 and 2 and has colour 1.
```
Comment (ReymondAkpanya) [inline gap/ColouredComplexes/edgeColouring.gd:331]: For example edge 1 is

File: gap/ColouredComplexes/isoscelesColouring_images.gd [hunk 2]
```diff
@@ -61,18 +61,14 @@ DrawSurfaceToTikz(colOct, "Oct_isosccoloured", pr );;
...
-#! @EndExampleSession
-#! This subdivided surface be drawn with <K>DrawSurfaceToTikz</K> 
-#! (<Ref Subsect="DrawSurfaceToTikz_EdgeColoured"/>).
-#! @BeginLog
+#! @BeginExample
 oct := Octahedron();;
 colOct := EdgeColouredPolygonalComplex( oct, [1,1,1,1,2,2,1,2,1,2,1,1]);;
 wildOct := WildColouredSurfaceOfIsoscelesColouredSurface(colOct);;
+#! @EndExample
+#! This subdivided surface be drawn with <K>DrawSurfaceToTikz</K>
```
Comment (ReymondAkpanya) [inline gap/ColouredComplexes/isoscelesColouring_images.gd:69]: This subdivided surface can be ....

**Review by ReymondAkpanya — CHANGES_REQUESTED** [review]

File: gap/ColouredComplexes/isoscelesColouring.gi [hunk 3]
```diff
@@ -477,37 +481,25 @@ InstallMethod( LocalSymmetryOfEdges,
...
 ##
 ## This function determines all edge colourings of a simplicial surface such
 ## that two edges have the same colour, representing isosceles triangles. Note
 ## that not all isosceles colourings are obtained by setting two colours
 ## equal in a wild colouring.
 ##
-
 InstallOtherMethod( AllIsoscelesColouredSurfaces, "for a simplicial surface",
-    [IsSimplicialSurface,IsList],
-            function( surf, li )
```
Comment (TomGoertzen) [inline gap/ColouredComplexes/isoscelesColouring.gi:510]: what was the purpose of li in the old version of the method?

**Review by TomGoertzen — COMMENTED** [review]
Very nice! I would suggest some small changes in the gd and I have a question about AllIsosc.... (see above)

File: gap/ColouredComplexes/isoscelesColouring.gi (hunk 3 above)
Comment (MeikeWeiss) [reply gap/ColouredComplexes/isoscelesColouring.gi:510]: Since li wasn't use in the old implementation, I think there wasn't a purpose.

Comment (TomGoertzen) [reply gap/ColouredComplexes/isoscelesColouring.gi:510]: ah ok that's funny :)! thank you

**Review by ReymondAkpanya — APPROVED** [review]

**Pull request merged**